develop/devguide/Automation/Howto/Creating_a_cypress_test_for_an_interactive_automation_script.md

```diff
@@ -0,0 +1,245 @@
+             
+             // Starts the event loop and shows the first dialog.
+             controller.ShowDialog(helloWorldDialog);
+          }
+          catch (ScriptAbortException)
+          {
+             // Catch normal abort exceptions (engine.ExitFail or engine.ExitSuccess)
+             throw; // Comment if it should be treated as a normal exit of the script.
+          }
+          catch (ScriptForceAbortException)
+          {
+             // Catch forced abort exceptions, caused via external maintenance messages.
+             throw;
+          }
+          catch (ScriptTimeoutException)
+          {
+             // Catch timeout exceptions for when a script has been running for too long.
+             throw;
+          }
+          catch (InteractiveUserDetachedException)
+          {
+             // Catch a user detaching from the interactive script by closing the window.
+             // Only applicable for interactive scripts, can be removed for non-interactive scripts.
+             throw;
+          }
+          catch (Exception e)
+          {
+             engine.ExitFail($"Run|Something went wrong: {e}");
+          }
+       }
+   }
+
+   public class HelloWorldDialog : Dialog
+   {
+      public HelloWorldDialog(IEngine engine) : base(engine)
+      {
+          // Create a label widget
+          var label = new Label("Hello, World!") { Style = TextStyle.Title };
+          
+          // Define the debug tag to be used in the cypress test
+          label.DebugTag = "label";
+          
+          // Add it to the dialog grid at position 0,0
+          AddWidget(label, 0, 0);
+          
+          // Create a button widget
+          var button = new Button("OK");
+          
+          // Define the debug tag to be used in the cypress test
+          button.DebugTag = "button";
+          
+          // Add it to the dialog just below the label
+          AddWidget(button, 1, 0);
+          
+          // Add an action to be performed whenever the button is clicked
+          button.Pressed += (sender, args) => engine.ExitSuccess("Done");
+      }
+    }
+    ```
+
+## Setting up the web environment
+
+1. [Create a new low-code app](xref:Creating_custom_apps).
+
+1. Add a button to the low-code app and configure it to trigger the created interactive Automation script when clicked.
+
+   For more information, see [Button](xref:DashboardButton).
+
+## Creating the Cypress test
+
+This Cypress test demonstrates how to use the *DebugTag* defined in an Automation script to verify that the text displayed on a label and button is correct.
+
+> [!NOTE]
+> Ensure that all *DebugTag* values used in your Automation script are unique per IAS page. This allows you to reference widgets unambiguously during testing. For example, by setting the *DebugTag* for a label to *myLabel*, you can locate the widget in the test using the following code:
+>
+> ```javascript
+> cy.get('[data-cy="myLabel"]');
+> ```
```

Comment:
Thank you very much for your suggestions!

I believe it would be better to add this information in the "Creating the Cypress Test" section, since it already provides a Cypress code example and an explanation.
That way, it would make it easier to read and understand the Cypress code.

@slc-peter-vanpoucke, what do you think?

Reply:
@MariekeGO @AndreRodriguesSkyline I agree that it's better to add it in the "Creating the Cypress Test" section. So fine for me to combine the info in the note on line 181/182.